README.md

```diff
@@ -1,4 +1,4 @@
 DeathKnell
 ==========
 
-A class that allows a final callback when fatal errors (SEGV, SIGKILL, etc) are received
+A unit test helper class that allows a final callback when fatal errors (SEGV, SIGKILL, etc) are received.  This is for **unit testing of death scenarios**. It should **not** currently be used in production code. 
```

Comment:
I thought it worked but I was wrong. 

I've been going back and forth on this one.  I see two options. 
1) Adjust vrecan/g2log-dev (enable a hidden API) and correct the Death library.
2) Since Death has never worked in production code, just remove it from the production code and keep it to the test code where it works fine. 

At startup the big 3 calls ``` Death::EnableDefaultFatalCall(g2logger)```. This does not call ```Death::SetupExitHandler``` which is needed for it to work in production code. 

```void Death::SetupExitHandler() {
   g2::internal::changeFatalInitHandlerForUnitTesting(Death::Received);
} ```

The other Death usage in QueueNado is to use ```Death::RegisterDeathEvent
``` to clean up IPC files. And in the big 3s to use the worker thread to exit quickly ```Death::WasKilled```


The 'hook-ins' however would require that ``` Death::EnableDefaultFatalCall(g2logger);``` is not enabled and instead